packages/components/alert/src/alert-icons.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import {
+  CloseIcon,
+  DangerIcon,
+  InfoCircleIcon,
+  SuccessIcon,
+  WarningIcon,
+} from "@nextui-org/shared-icons";
+
+export const AlertCloseIcon = (props: {color: string}) => {
+  switch (props.color) {
+    case "primary":
+      return <CloseIcon className={`fill-current text-primary-200`} height={20} width={20} />;
+
+    case "secondary":
+      return <CloseIcon className={`fill-current text-secondary-200`} height={20} width={20} />;
+
+    case "success":
+      return <CloseIcon className={`fill-current text-success-200`} height={20} width={20} />;
+
+    case "warning":
+      return <CloseIcon className={`fill-current text-warning-200`} height={20} width={20} />;
+
+    case "danger":
+      return <CloseIcon className={`fill-current text-danger-200`} height={20} width={20} />;
+
+    default:
+      return <CloseIcon className={`fill-current text-default-400`} height={20} width={20} />;
```

Comment:
there is no variable used in className. You can simply use string. e.g.

```
className="fill-current text-primary-200"
```

Reply:
Must have forgot to change that from my previous code. will make changes.